tests/check_clvm.py

```diff
@@ -0,0 +1,166 @@
+from __future__ import annotations
+
+import dataclasses
+import os
+import pathlib
+import sys
+import tempfile
+import traceback
+import typing
+
+here = pathlib.Path(__file__).parent.resolve()
+root = here.parent
+
+# This is a work-around for fixing imports so they get the appropriate top level
+# packages instead of those of the same name in the same directory as this program.
+# This undoes the Python mis-feature meant to support 'scripts' that have not been
+# installed by adding the script's directory to the import search path.  This is why
+# it is simpler to just have all code get installed and all things you run be
+# accessible via entry points.
+sys.path = [path for path in sys.path if path != os.fspath(here)]
+
+from clvm_tools_rs import compile_clvm  # noqa: E402
+
+from chia.types.blockchain_format.program import SerializedProgram  # noqa: E402
+
+clvm_suffix = ".clvm"
+hex_suffix = ".clvm.hex"
+hash_suffix = ".clvm.hex.sha256tree"
+
+
+def generate_hash_bytes(hex_bytes: bytes) -> bytes:
+    cleaned_blob = bytes.fromhex(hex_bytes.decode("utf-8"))
+    serialize_program = SerializedProgram.from_bytes(cleaned_blob)
+    result = serialize_program.get_tree_hash().hex()
+    return (result + "\n").encode("utf-8")
+
+
+@typing.final
```

Comment:
isn't this a Py3.8+ feature?

Reply:
Yes, that should come from `typing_extensions`...  Thanks.  Usually tests catch that, but this makes me feel like we're missing some sort of check.